src/app/base/components/PrefixedIpInput/PrefixedIpInput.tsx

```diff
@@ -67,14 +67,18 @@ const PrefixedIpInput = ({ cidr, name, ...props }: Props) => {
   return (
     <PrefixedInput
       help={
-        subnetIsIpv4 ? (
-          <>
-            The available range in this subnet is{" "}
-            <code>
-              {immutable}.{editable}
-            </code>
-          </>
-        ) : null
+        !help ? (
+          subnetIsIpv4 ? (
+            <>
+              The available range in this subnet is{" "}
+              <code>
+                {immutable}.{editable}
+              </code>
+            </>
+          ) : null
+        ) : (
+          help
```

Comment:
I don't know what the linter would do but `help ? help : ( ...` might be simpler to read.

Reply:
Yeah, that does sound simpler. Will udpate